zerver/tests/test_push_notifications.py

```diff
@@ -704,11 +704,40 @@ def test_get_apns_payload(self):
             'custom': {
                 'zulip': {
                     'message_ids': [message.id],
+                    'recipient_type': 'private',
+                    'sender_email': 'hamlet@zulip.com',
                 }
             }
         }
         self.assertDictEqual(payload, expected)
 
+    def test_get_apns_payload_stream(self):
+        # type: () -> None
+        stream = Stream.objects.filter(name='Verona').get()
+        message = self.get_message(Recipient.STREAM, stream.id)
+        message.triggers = {
+            'private_message': False,
+            'mentioned': True,
+            'stream_push_notify': False,
+        }
```

Comment:
You'll want to change the `trigger` format after rebasing on top of e98ca0714b36e92011488a9879909e9ccbbeeeba

Reply:
Yeah will change the trigger here as well!